nextcloudappstore/core/models.py

```diff
@@ -88,34 +89,52 @@ def can_update(self, user: User) -> bool:
     def can_delete(self, user: User) -> bool:
         return self.owner == user
 
-    def releases_by_platform_v(self):
+    def releases_by_platform_v(self, nightlies=False):
```

Comment:
exclusive selection? To me it looks like using releases_by_platform(True) would also include nightlies, am i wrong?